src/parser/demonhunter/havoc/modules/talents/BlindFury.js

```diff
@@ -53,7 +54,7 @@ class BlindFury extends Analyzer{
         return suggest(<>Try to cast <SpellLink id={SPELLS.EYE_BEAM.id} /> at 30 to 50 fury. Having more than 50 fury at cast is considered a dps loss.</>)
           .icon(SPELLS.BLIND_FURY_TALENT.icon)
           .actual(`${actual} bad casts`)
-          .recommended(`0 is recommended.`);
+          .recommended(`${formatPercentage(recommended)}% is recommended.`);
```

Comment:
Technically the suggestion threshold is in terms of cast number rather than %. But 0 and 0% are the same thing so not a big issue unless the `minor` threshold were increased.

Reply:
0 bad casts will always be the recommended